lib/components/user/new-account-wizard.js

```diff
@@ -5,7 +5,9 @@ import SequentialPaneDisplay from './sequential-pane-display'
 /**
  * This component is the new account wizard.
  */
-const NewAccountWizard = ({ onComplete, panes, userData }) => {
+const NewAccountWizard = props => {
+  const { panes, values: userData } = props
```

Comment:
I don't fully understand the nature of this change. It would be helpful to have some comments about what is contained within `props` that the individual panes are receiving.

Reply:
Addressed in https://github.com/opentripplanner/otp-react-redux/pull/239/commits/fb18f75ff1e1417ab6e8a359b5091fba4cdcc6d3